WordPress/Sniffs/PHP/StrictComparisonsSniff.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Enforces Strict Comparison checks, based upon Squiz code
+ *
+ * PHP version 5
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @author   Matt Robinson
+ */
+
+class WordPress_Sniffs_PHP_StrictComparisonsSniff implements PHP_CodeSniffer_Sniff
+{
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register()
+	{
+		return array(
+			T_IS_EQUAL,
+			T_IS_NOT_EQUAL,
+		);
+
+	}//end register()
+
+
+	/**
+	 * Processes this test, when one of its tokens is encountered.
+	 *
+	 * @param PHP_CodeSniffer_File	$phpcsFile The file being scanned.
+	 * @param int			$stackPtr  The position of the current token in the
+	 *						stack passed in $tokens.
+	 *
+	 * @return void
+	 */
+	public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
+	{
+		$tokens = $phpcsFile->getTokens();
+
+		if ( $tokens[$stackPtr]['code'] !== T_IS_EQUAL && $tokens[$stackPtr]['code'] !== T_IS_NOT_EQUAL) {
```

Comment:
I don't think that this condition is necessary. PHPCS will already be limiting the sniff to these tokens based on `register()`, won't it?